builtin/common/misc_helpers.lua

```diff
@@ -559,6 +559,10 @@ if INIT == "mainmenu" then
 
 		return nil
 	end
+
+	function get_modpack_path(mod)
```

Comment:
Undocumented global function.
Workaround: Move it to the other functions in `mainmenu/common.lua`, as it's not helpful to mods.